onnxruntime/python/onnxruntime_pybind_mlvalue.cc

```diff
@@ -108,11 +108,19 @@ void CreateTensorMLValue(AllocatorPtr alloc, const std::string& name_input, PyAr
     // numpy requires long int as its dims.
     int ndim = PyArray_NDIM(darray);
     npy_intp* npy_dims = PyArray_DIMS(darray);
-    std::vector<int64_t> dims(ndim);
-    for (int i = 0; i < ndim; ++i) {
-      dims[i] = npy_dims[i];
+    std::vector<int64_t> dims;
+    // numpy arrays created using this syntax "np.array(['4'])" have a shape of (1,). This doesn't match the
```

Comment:
not understanding this. isn't the syntax correct? would this be an issue with the test or its model/conversion?